Formula/xcftools.rb

```diff
@@ -0,0 +1,59 @@
+class Xcftools < Formula
+  desc "Tools for GIMP's native file format XCF"
+  homepage "http://henning.makholm.net/software"
+  url "http://henning.makholm.net/xcftools/xcftools-1.0.7.tar.gz"
+  sha256 "1ebf6d8405348600bc551712d9e4f7c33cc83e416804709f68d0700afde920a6"
+
+  depends_on "libpng"
+
+  def install
+    # Apply patch to build with libpng-1.5 or above
+    # http://anonscm.debian.org/cgit/collab-maint/xcftools.git/commit/?id=c40088b82c6a788792aae4068ddc8458de313a9b
```

Comment:
As mentioned in the previous comment, already sent an email to the author about the patches, but didn't get a response yet. There is no forum or bug tracker that accepts patches.